src/OrbitGl/UIElement.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_UI_ELEMENT_H_
+#define ORBIT_GL_UI_ELEMENT_H_
+
+#include "PickingManager.h"
+
+class TimeGraph;
+class GlCanvas;
+
+namespace orbit_gl {
+
+/* Base class for UI elements drawn underneath the capture window. */
+class UIElement : public Pickable {
```

Comment:
By the way, according to the style guide abbreviations should be capitalized as normal words, i.e., `UiElement` (but see the other comment on being more specific with this name).